### ISSUE TITLE
Ensure consistent evaluation perspective across the code base

### ISSUE THREAD

**Issue by knutole** [py-stockfish/stockfish#18]
There are three ways to return evaluations: 
- From the perspective of whose turn it is (ie. `+5` is advantage to the side whose turn it is)
- From the perspective of White always (ie. `+5` is advantage White, no matter whose turn it is)
- From the perspective of Black always (ie. `+5` is advantage Black, no matter whose turn it is)

Currently there is a happy mix in the codebase:
- Stockfish engine uses Turn perspective.
- [get_evaluation()](https://github.com/py-stockfish/stockfish/blob/master/stockfish/models.py#L506) returns White perspective always 
- [get_top_moves() ](https://github.com/py-stockfish/stockfish/pull/16/files#diff-04c95822152cdc47e4a5a4f1a811eb88ddd1935de9011dd908083878e5ea3864R650)returns White perspective by default (but uses new [`is_turn_perspective`](https://github.com/py-stockfish/stockfish/pull/16/files#diff-04c95822152cdc47e4a5a4f1a811eb88ddd1935de9011dd908083878e5ea3864R33) option)
- [get_wdl_stats()](https://github.com/py-stockfish/stockfish/blob/master/stockfish/models.py#L442) returns Turn perspective

Suggested solution: 
- Use `is_turn_perspective` option for all functions, default to `True` (ie. Turn perspective).
- Problem: backwards compatibility for above functions that use White perspective.

**Comment by kieferro** [comment]
Good point. We should definitely try to standardize that. If there's no other way, we could also consider deviating a bit from backwards compatibility and doing a major version bump with bigger changes.
